src/ray/pubsub/subscriber.cc

```diff
@@ -92,7 +94,9 @@ SubscriptionCallback SubscriberChannel<KeyIdType>::GetCallbackForPubMessage(
                  << publisher_id;
 
   auto maybe_subscription_callback = GetSubscriptionCallback(publisher_address, key_id);
+  cum_published_messages_++;
```

Comment:
how about this: make this one mutable, and then you can put const back?

Reply:
Sounds like a good idea!